tests/Unit/PointwiseFunctions/MathFunctions/TestMathHelpers.hpp

```diff
@@ -0,0 +1,30 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Contains helper functions for testing math functions
+
+#pragma once
+
+#include <random>
+
+#include "tests/Unit/TestHelpers.hpp"
+
+///
```

Comment:
I'm not sure you had this almost blank line here intentionally, so I just want to point it out.